smsdk/client.py

```diff
@@ -682,13 +778,20 @@ def get_machine_types(self, source_type=None, *args, **kwargs):
 
         return mts
 
-    def get_machine_type_names(self, clean_strings_out=True):
+    def get_machine_type_names(self, *args, **kwargs):
```

Comment:
Could have left clean_strings_out=True

Reply:
This is the requirement of the Jira item. The SDK-API functions should be transformed from their current parameter format to a more SQL-like structure.